internal/config/config_test.go

```diff
@@ -0,0 +1,23 @@
+//go:build all
+// +build all
+
+package config
+
+import "testing"
+
+func TestNewConfig(t *testing.T) {
```

Comment:
Please think about how you should test this function again. Current implementation is not really testing anything and will depend on the system state (such as environment variables set in the active system) instead of testing for expected behavior.
- Prefer table tests, as with examples I've provided yesterday.
- consider setup and tear-down steps for testing how config initialization behaves with different ENV variables.

Reply:
i made some changes, is that correct - setting env vars while testing?

Reply:
you should use `t.SetEnv` instead of `os.SetEnv` in tests.

Reply:
fixed